custom_components/ltss/__init__.py

```diff
@@ -199,9 +215,11 @@ def notify_hass_started(event):
 
             tries = 1
             updated = False
-            while not updated and tries <= 10:
+            while not updated and (
+                self.db_retry_limit is None or tries <= self.db_retry_limit
+            ):
```

Comment:
We cannot allow this part to be configurable by the end user. If we loose connectivity with the database while in normal operation, we cant retry indefinitely until the database comes back online as this behavior will fill the internal queue but we will not discard any old states (due to the indefinite retrials) ... 

Also, since we managed to get a working connection during startup of the component, sqlalchemy will handle reconnections automatically.

Reply:
Ok, I removed this chunk. It's as was before - hardcoded limit of 10 retries, and relying on SQLAlchemy to reconnect.